CoreGraphFile/src/au/gov/asd/tac/constellation/graph/file/io/GraphJsonReader.java

```diff
@@ -100,15 +100,20 @@ public GraphJsonReader() {
     public Graph readGraphZip(final File graphFile, final IoProgress progress) throws IOException, GraphParseException {
         try (final InputStream in = new BufferedInputStream(new FileInputStream(graphFile))) {
             return readGraphZip(graphFile.getPath(), in, progress);
+            
         }
     }
 
     public Graph readGraphZip(final String name, InputStream bin, final IoProgress progress) throws IOException, GraphParseException {
         try (bin) {
             progress.start(100);
-            progress.progress("Reading file: " + name);
-            
             byteReader = new GraphByteReader(bin);
+        } 
+        catch (IOException ex) {
```

Comment:
final

Reply:
addressed, also checked for others in file.